JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,19 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  if (seconds > 0) {
+    let secondsT = seconds % 60;
+    let hoursT = Math.floor((seconds - secondsT) / 60 / 60);
+    let minutesT = (seconds - secondsT) / 60 - 60 * hoursT;
+
+    const hoursS = hoursT > 9 ? `${hoursT}` : `0${hoursT}`;
```

Comment:
the template string for `${hoursT}` is unnecessary, you can leave it as `hoursT`

Reply:
lines 21, 22, 23 are doing the same operation, you should find a way to avoid duplicating the same code